Cargo.toml

```diff
@@ -15,9 +15,20 @@ edition = "2018"
 members = ["benches"]
 exclude = ["test-apps", "testing-tools"]
 
+[[example]]
+name = "ast"
+
+[[example]]
+name = "stats"
+
+[[example]]
+name = "print_pos"
+required-features = ["token-ranges"]
```

Comment:
I don't think we need this anymore.

Or at least we should simply feature gate the main in the example file.

Reply:
Without this, `cargo test --no-default-features` would fail. I therefore feature-gated the main function instead of listing the examples.